data/hypervisor.args.in

```diff
@@ -2,7 +2,7 @@
 -name
 @NAME@
 -machine
-pc-lite,accel=kvm,kernel_irqchip,nvdimm
+pc,accel=kvm,kernel_irqchip,nvdimm
```

Comment:
I've  just noticed that you're not trying to merge this into `master`. As long as we're all clear that these changes won't land in `master` until we've investigated their impact further, that's fine.